Sources/SwiftLayout/Layoutable/Wrapper/SL.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SL.swift
+//  
+//
+//  Created by oozoofrog on 2022/03/13.
+//
+
+import Foundation
+
+public final class SL<L: Layoutable> {
```

Comment:
In fact, the name "SL" is ambiguous class name.
It's a bit long, but I think you need a clear name like "LayoutableMethodWrapper"